pages/admin/admin-management.tsx

```diff
@@ -142,12 +143,53 @@ export default function AdminManagement() {
     },
   ];
 
+  // Placeholder data
+  const DATA = [
+    {
+      name: 'Steve Rogers',
+      email: 'steverogers@uwblueprint.org',
+      role: Role.Secretary,
+    },
+    {
+      name: 'Tony Stark',
+      email: 'tstark@avengers.inc',
+      role: Role.Accounting,
+    },
+    {
+      name: 'Hulk',
+      email: 'incrediblehulk@smash.com',
+      role: Role.Admin,
+    },
+    {
+      name: 'Doctor Strange',
+      email: 'strange@uwblueprint.org',
+      role: Role.Secretary,
+    },
+    {
+      name: 'Spiderman',
+      email: 'spider@avengers.inc',
+      role: Role.Accounting,
+    },
+    {
+      name: 'Thor',
+      email: 'thegodthor@asgard.odin',
+      role: Role.Admin,
+    },
+  ];
```

Comment:
Not sure why we needed to move the sample data inside the component but it's no big deal